text-ui-test/ACTUAL.TXT

```diff
@@ -0,0 +1,12 @@
+Hello! I'm Duke
```

Comment:
Detailed actual output showcasing different commands of Duke. Could include error responses.